learn/nginx.conf

```diff
@@ -0,0 +1,66 @@
+user user user;
+worker_processes 1;
+error_log /var/log/nginx/error.log;
+pid /dev/null;
+daemon off;
+
+# Load dynamic modules. See /usr/share/nginx/README.dynamic.
+include /usr/share/nginx/modules/*.conf;
+
+env TEST_DATA=/cache;
+env HOME=/home/user;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
```

Comment:
Same here.

Reply:
sadly the http module only accepts files, so we need the symlink here